src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -2,14 +2,26 @@
     <div
         id="test-accordion-item-container-{{ i }}"
         class="pointer"
-        [ngClass]="{ 'd-none': !(groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }"
+        [ngClass]="{ 'd-none': sidebarType !== 'conversation' && !(groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }"
     >
         <div id="test-accordion-item-header" class="control-label d-flex align-items-center justify-content-between py-2 bg-module" (click)="toggleGroupCategoryCollapse(groupKey)">
             <div class="my-2">
-                {{ 'artemisApp.courseOverview.sidebar.' + groupKey | artemisTranslate | titlecase }}
+                @if (sidebarType !== 'conversation') {
+                    {{ 'artemisApp.courseOverview.sidebar.' + groupKey | artemisTranslate | titlecase }}
+                } @else {
+                    <fa-icon [fixedWidth]="true" [icon]="getIcon(groupKey)" />
+                    {{ 'artemisApp.conversationsLayout.conversationSelectionSideBar.' + groupKey | artemisTranslate }}
+                }
                 ({{ (groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }})
             </div>
-            <div class="pe-3">
+            <div class="icon-container pe-3">
+                <jhi-accordion-add-options
```

Comment:
Same here. We should introduce something like `showMoreOptionsDropdown` and provide the desired data at this point to the new component. This way, we also avoid the add-options-component being specifically designed for the conversations.